components/channel_layout/center_channel/index.ts

```diff
@@ -45,6 +45,7 @@ const mapStateToProps = (state: GlobalState, ownProps: Props) => {
         lhsOpen: getIsLhsOpen(state),
         rhsOpen: getIsRhsOpen(state),
         rhsMenuOpen: getIsRhsMenuOpen(state),
+        isMobile: state.views.channel.mobileView,
```

Comment:
would we use `getIsMobileView` selector? 0/5 on it thought

Reply:
+1 to that

Reply:
Yep, good call. I'll update it.